_scripts/instruction-widget/templates/install/snap.html

```diff
@@ -53,6 +53,11 @@ <h3>Snap Support</h3>
 
 {{^advanced}}
```

Comment:
Since this is enclosed in `{{^advanced}}` it doesn't show up in our wildcard instructions which I think should be fixed.

Reply:
Fixed.